scripts/config/templates/nginx.conf.erb

```diff
@@ -246,8 +242,8 @@ http {
   }
 
   # The /build directory is also needed because that's where the server-rendered JS comes from
-  # Redirect location DLPs to server-side rendered frontend
-  location ~ ^/(([0-9A-z\-]*)-near-me\/((indoor|outdoor|covered)\/)?((?!(indoor|outdoor|covered))[A-z\-]+)\/([A-z\-]+)$) {
+  # Proxy location DLPs to server-side rendered frontend
+  location ~ ^/(([0-9A-z\-]*)-near-me\/((indoor|outdoor|covered)\/)?((?!(indoor|outdoor|covered))[A-z\-]+)\/([A-zÀ-ÖØ-öø-ÿ\-\']+)$) {
     proxy_pass <%= ssr_frontend_host %>/$1$is_args$args;
```

Comment:
For some reason, the url that is getting proxied is decoding the chars wrong, resulting in a render that looks like:
![Screenshot from 2022-05-24 17-28-31](https://user-images.githubusercontent.com/13811962/170148104-741b812c-6b9c-408c-b73b-54667d4238e1.png)